output.go

```diff
@@ -65,10 +65,13 @@ func outputLineAsHTML(line []node) string {
 				}
 			}
 		}
-		if node.elem != nil {
-			lineBuf.buf.WriteString(node.elem.asHTML())
-		} else {
-			lineBuf.appendChar(node.blob)
+
+		if elem := node.elem; elem != nil {
+			lineBuf.buf.WriteString(elem.asHTML())
+		}
+
+		if r, ok := node.getRune(); ok {
+			lineBuf.appendChar(r)
```

Comment:
I didn't like the implied but non-obvious “if it doesn't have an element it should have its rune written”, so I bumped that up to a conditional `getRune()` method at least. But keeping the change minimal for now.